src/components/MainForm/Mainform.tsx

```diff
@@ -1,35 +1,137 @@
 import React, { useState } from "react";
 import Header from "../Header/Header";
-import "./MainForm.css";
-//Material-UI
+import { makeStyles } from "@mui/styles";
+import { Theme } from "@mui/material";
 import TextField from "@mui/material/TextField";
 import MenuItem from "@mui/material/MenuItem";
 import Box from "@mui/material/Box";
 import Button from "@mui/material/Button";
 import Dialog from "@mui/material/Dialog";
-import DialogActions from "@mui/material/DialogActions";
 import DialogContent from "@mui/material/DialogContent";
 import DialogTitle from "@mui/material/DialogTitle";
-import InputLabel from "@mui/material/InputLabel";
 import FormControl from "@mui/material/FormControl";
-import Select, { SelectChangeEvent } from "@mui/material/Select";
-
-//React Hook Form
 import { useForm, SubmitHandler } from "react-hook-form";
-// const countries = [
-//   {
-//     value: "USD",
-//     label: "$",
-//   },
-//   {
-//     value: "AUS",
-//     label: "A$",
-//   },
-//   {
-//     value: "PAK",
-//     label: "PKR",
-//   },
-// ];
+import { useHistory } from "react-router-dom";
+import store from "../../store/store";
+const countries: any = {
+  USA: {
+    value: "+1-USA",
+    icon: "images/united-states.png",
+  },
+  AUS: {
+    value: "+61-AUS",
+    icon: "images/australia.png",
+  },
+  PAK: {
+    value: "+92-PAK",
+    icon: "images/pakistan.png",
+  },
+};
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    height: "100vh",
+    width: "20%",
+    borderRadius: "5px",
+    padding: "20px",
+    display: "flex",
+    alignItems: "center",
+    margin: "auto",
+  },
+  iconifyMain: {
+    height: "100%",
+    width: "100%",
+  },
+  iconifyMenu: {
+    height: "160%",
+    width: "30%",
+    position: "absolute",
+    left: "-13px",
+  },
+  iconifyMenuUS: {
+    width: "30%",
+    position: "absolute",
+    left: "-12px",
+    height: "90%",
+    borderRadius: "11px",
+  },
+  iconifyButtons: {
+    marginBottom: "7%",
+  },
+  phoneNumber: {
+    width: "80%",
+    margin: "0 0 20px 20% !important",
+    display: "flex",
+  },
+  errorMsg: {
+    color: "red",
+    fontSize: "small",
+    fontFamily: "Raleway, sans-serif",
```

Comment:
you dont need this class , as you can use helper text for error messages.

Reply:
done